.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: boilerplate
   namespace: openshift
-  tag: image-v6.0.1
+  tag: image-v7.0.0
```

Comment:
I know this is just a draft, but we should update the operator to use Go 1.23 as well, since that is what comes in this CI image.